### PR TITLE
sync with 2.7.0

### PR DESCRIPTION
**Pull request by Rumperuu**
Last time

### CODE REVIEW

**Comment by markcheret** [comment]
holy moly, quite the divergence! Thank you for merging it. I couldn't do it!

File: class/dashboard/subpage-main.php [hunk 1]
```diff
@@ -178,13 +178,13 @@ public function amp_compat() {
 		// Replace all placeholders.
 		$l_obj_template->replace(
 			array(
-				// Translators: '%s' is the link text 'AMP-WP' linked to the plugin's front page on WordPress.org.
+				// Translators: '%s' is the link text 'AMP-WP' linked to the plugin’s front page on WordPress.org.
```
Comment (Rumperuu) [inline class/dashboard/subpage-main.php:181]: @pewgeuges please fix the apostrophes etc.

**Review by markcheret — APPROVED** [review]
What a massive divergence. Holy cow. Thanks for syncing it up!!!

**Comment by Rumperuu** [comment]
> What a massive divergence. Holy cow. Thanks for syncing it up!!!

This is what I meant by `git svn rebase` being effectively unusable now, because the two codebases have been diverging for ~3 weeks and 13 versions now. Every time you run the command, it starts way back in the commit history and you have to deal with each merge conflict in turn, and then merge `main` with whatever you're left with to try and make sure you've got everything from both in and you're not going to accidentally undo changes made here because they haven't been propagated to the SVN repo.

So I'm definitely not slogging through another of these.

**Comment by markcheret** [comment]
Yes, so I'm just going to do a plain DIFF between SVN and Git now and just take the recent bugfix on board and then do the release testing.

**Pull request merged**